### ISSUE TITLE
N50 and L50 

### ISSUE THREAD

**Issue by cabraham03** [sanger-pathogens/assembly-stats#14]
just one question, for assembly-stats example (in your main page),  in N50=1687656 and n=5, n correspond to L50 ???
Thanks 

$ assembly-stats Pf3D7_v3.fasta
stats for Pf3D7_v3.fasta
sum = 23328019, n = 16, ave = 1458001.19, largest = 3291936
N50 = 1687656, n = 5
N60 = 1472805, n = 7
N70 = 1445207, n = 8
N80 = 1343557, n = 10
N90 = 1067971, n = 12
N100 = 5967, n = 16
N_count = 0
Gaps = 0

**Comment by martinghunt** [comment]
Yes, L50 = 5 in that example. All the  n=X numbers are the L50/60/.../100 values corresponding to the N50/60/...100 values.
